packages/fhir-sanitize-param/index.test.js

```diff
@@ -118,12 +118,6 @@ describe('Standard Parameter Sanitization Tests', () => {
       };
       expect(callSanitizeNumber).toThrowError(/expected number for parameter foo/);
     });
-    test('Should reject invalid numbers', () => {
```

Comment:
Was this test not needed anymore?

Reply:
the issue that this is testing was fixed by a dep update